Kbs.Business/User/UserValidator.cs

```diff
@@ -6,7 +6,7 @@ namespace Kbs.Business.User;
 public class UserValidator
 {
     private static readonly Regex EmailValidationRegex =
-        new Regex("^[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?$", RegexOptions.Compiled);
+        new Regex("^[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?\\.)+[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?$", RegexOptions.Compiled);
```

Comment:
Should be obselete, [a-9] should be the same as [a-zA-Z0-9]

Reply:
> Should be redundant, [a-9] should be the same as [a-zA-Z0-9]

this is incorrect. a-9 provides an invalid pattern as the character '9' is of lower value in Unicode/ASCII than character 'a'